### PR TITLE
Fix/cropper

### PR DESCRIPTION
**Pull request by Alex-Craftsman**

### CODE REVIEW

File: Form/Type/DropzoneType.php [hunk 1]
```diff
@@ -120,11 +123,11 @@ public function configureOptions(OptionsResolver $resolver)
                 'multiple' => 'autodetect',
                 'maxFiles' => static::DEFAULT_MAX_FILES,
                 'maxFileSize' => static::DEFAULT_MAX_SIZE,
-                'type' => 'form_widget',
+                'type' => $fieldType,
```
Comment (webbertakken) [inline Form/Type/DropzoneType.php]: So a default should never be variable. In the form where this FormType is being used, it should be used as configuration

File: Form/Type/DropzoneType.php [hunk 2]
```diff
@@ -120,11 +123,11 @@ public function configureOptions(OptionsResolver $resolver)
...
                 'btnClass' => 'btn btn-info btn-lg',
                 'btnText' => 'Files',
                 'uploaderText' => 'Drop files here to upload',
-                'style_type' => 'style_default',
+                'style_type' => 'style_cropper',
```
Comment (webbertakken) [inline Form/Type/DropzoneType.php]: The defaults resolver should not be used to globally configure the FormType. Instead it should be the default style (having another style called 'style_simple' that holds the old default behaviour) or configured in the form itself.

File: Resources/views/Form/_common.html.twig [hunk 3]
```diff
@@ -192,16 +191,20 @@
...
               });
 
               $('#{{ form.vars.id }}').on('click', '.js-open-cropper-modal', function(e) {
                 e.preventDefault();
                 var fileName = $(this).data('file-name');
-                var removeFileName = $(this).data('remove-file-name');
-                var url = location.protocol + '//' + location.hostname + '/uploads/gallery/' + fileName;
-                var altUrl = location.protocol + '//' + location.hostname + '/upload/files/get/' + removeFileName;
+                var origFileName = $(this).data('orig-file-name');
+
```
Comment (webbertakken) [inline Resources/views/Form/_common.html.twig]: useless empty line without comment.

File: Resources/views/Form/_common.html.twig [hunk 4]
```diff
@@ -282,6 +285,10 @@
             },
           }
         );
-      });
+      })
+    {% if form.vars.type == 'crop' %}
+    )
+    {% endif %}
+    ;
```
Comment (webbertakken) [inline Resources/views/Form/_common.html.twig]: Could you maybe fix this generated syntax?

**Comment by Alex-Craftsman** [comment]
@webbertakken , please, check

**Pull request merged**